app/services/submission_service.rb

```diff
@@ -20,14 +20,11 @@ def create_submission(
       user =
         if gravity_user_id
           User.find_or_create_by!(gravity_user_id: gravity_user_id)
+        elsif contact_information?(submission_params)
+          User.anonymous
```

Comment:
**issue:** I prefer that we set the user to `nil` instead of assigning the anonymous user record to a submission. Are there any issues with having a `nil` user?